packages/editor/src/components/blocks/select/Select.tsx

```diff
@@ -40,14 +40,14 @@ export const SelectComponent: ComponentConfig<SelectProps> = {
     dynamicItemsLabel: {
       subsection: 'Dynamic Options',
       label: 'Dynamic Label',
-      type: 'text',
-      options: { placeholder: 'Enter fitting attribute' }
+      type: 'textBrowser',
+      options: { onlyAttributesVariable: true, placeholder: 'Enter attribute (or leave blank to select entire object)' }
```

Comment:
maybe you can also think about to implement the `hide` function. With it you can control if the option should be visible. E.g. in this case the label and value only makes sense if you have defined the options value.

Reply:
Yes I agree. I will try to implement this in the next PR 👍